src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -98,7 +98,7 @@ class NotebookLauncherContent extends Component {
 
       const cluster = await this.getCluster()
       if (!cluster) {
-        throw new Error('No clusters available')
+        throw new Error('You do not have access to run analyses on this workspace.')
```

Comment:
I searched through the error messages and we use "access" and "permission" seemingly interchangeably: "You do not have access to add data to this workspace" and "You do not have permission to edit this workspace".

@bradtaylor (and others) Is there a meaningful difference for users between having access or permission, beyond semantics? I propose we choose one (access or permission) and use that.

Reply:
Good catch! It seems like we mostly use access, and there's only two instances where we use permission (in Dashboard and data-utils) which leads me to believe it's likely not a meaningful difference, and we should stick with one, probably access.

Reply:
Access is a bit generic. You may not have access for a number of reasons. For example, you may not have access to any workspaces because Workbench is down. Permission is more specific. If you are told you don't have permission, you may be able to resolve it by asking someone to grant you permission.